CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 3.10)
 set(CMAKE_EXPORT_COMPILE_COMMANDS ON)   # For clang-tidy.
-
+set(BUILD_SHARED_LIBS ON)
```

Comment:
Why was this added?

Reply:
To prevent "Linking CXX shared library error; recompile with -fPIC"